docs/advanced-recipes/atom-creators.mdx

```diff
@@ -356,3 +356,95 @@ const ListOfThings = () => {
 ```
 
 <CodeSandbox id="ugkzm0" />
+
+## atomWithDebounce
+
+> `atomWithDebounce` helps with creating an atom where state set should be debounced.
+
+This util is useful for text search inputs, where you would like to call **functions in derived atoms only once** after waiting for a duration, instead of firing an action on every keystroke.
+
+```tsx
+import { atom } from 'jotai'
+import { SetStateAction } from 'react'
```

Comment:
jotai exports the one, which seems cleaner?
```suggestion
import type { SetStateAction } from 'jotai'
```

Reply:
Ah, wrong import. 

Amended to

`import { atom, SetStateAction } from "jotai";`